### ISSUE TITLE
Download hungs without any error if old download path does not exist

### ISSUE THREAD

**Issue by kapollo** [FaisalUmair/udemy-downloader-gui#331]
version 1.8 (also on 1.7)
Problem:
1. Connect removable storage
2. Select path on the removable storage as a download path
3. save settings, close Udeler, disconnect removable device
4. start Udeler download something

Result:
Download hangs, without an error

**Comment by FaisalUmair** [comment]
Yes that's correct.  It would prevent it from downloading unless you reset the path. I guess it is the expected behaviour, but displaying an error pointing the exact problem would be nice (Will work on it)
